types/browan/TBWaterleak/meta.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Water Leak Sensor, TBWL100, Tabs",
+  "name": "Tabs Water Leak Sensor, TBWL100",
```

Comment:
changes in name will need to be done manually, therefore coordination when merging this is needed